README.md

```diff
@@ -0,0 +1,54 @@
+# cvs-tsk-scheduled-operations
+A service for running tasks which need to happen periodically, triggered by Cloudwatch Events, rather than user operations.  
+
+#### Run AWS Lambda node functions locally
+- `npm install`
+- `BRANCH=local npm start`
+
+### Git Hooks
+
+Please set up the following prepush git hook in .git/hooks/pre-push
+
+```
+#!/bin/sh
+npm run prepush && git log -p | scanrepo
+
+```
+
```

Comment:
you can remove this section since we added husky

Reply:
still useful to leave in - husky will overwrite if they're present